tools/start_all_tmux.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is used to start all of the microservices
+# for the Lucida project by creating a tmux window for each
+# server in the background.
+# Example calls:
+# $./start_all_tmux.sh secure
+# $./start_all_tmux.sh
+# To attach to the tmux session
+# use the following command:
+# $tmux a -t lucida
+
+SESSION_NAME="lucida"
+
+# Check to see if we should run on http/ws (non-secure) or https/wss (secure)
+if [ "$1" == "secure" ]; then
+    echo "Enabling secure host"
+    # Getting the host IP address
+    export ASR_ADDR_PORT="wss://$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}'):8081"
```

Comment:
Yes, that's how it seems. Each service has a port that is hard-coded and used in multiple places.